src/main/java/org/gridsuite/actions/server/dto/FilterContingencyListAttributes.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.actions.server.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+/**
+ * @author Chamseddine benhamed <chamseddine.benhamed at rte-france.com>
+ */
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class FilterContingencyListAttributes {
```

Comment:
What is the difference with FilterContingencyList? Why 2 different classes?

Reply:
FilterContingencyListAttributes  is used as Body model for  the /filters-contingency-lists/{name}  PUT endpoint

Reply:
You don't answer the question, This 2 classes are identical and used for the same pupose so why you don't keep just one (FilterContingencyList)?